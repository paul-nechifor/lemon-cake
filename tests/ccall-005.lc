(= abs (~ (a) (ccall (dynsym 'libc.so' 'abs') a)))

(repr (abs 5))
(repr ' ')
(repr (abs -7))
(repr ' ')
(repr (abs -2147483647))
