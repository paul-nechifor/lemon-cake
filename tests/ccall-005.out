5' '7' '2147483647
